src/model/SVGFile.java

```diff
@@ -34,15 +43,34 @@ public void setXmlcontent(String xmlcontent) {
 		this.xmlcontent = xmlcontent;
 	}
 
+	
+	/*
+	 * save SVG file into files
+	 */
 	@Override
-	public boolean writeSVGFile(String content) {
-		// TODO Auto-generated method stub
-		return false;
+	public boolean writeSVGFile() {
+		file = new File(String.format(filnename, this.getXmlcontent()));
+		if(file.exists()){
```

Comment:
I thought that .exists() check if the object exist, here you define file just above, so it will always exists no ?
Have you tested it ?